### ISSUE TITLE
extension mongodb failed compile check

### ISSUE THREAD

**Issue by MrZ9001** [crazywhalecc/static-php-cli#219]
When using php-8.2 to generate static php8.0 on Ubuntu 20.04, the debug log displays "extension mongodb failed compile check".
mongodb extension version: php_mongodb-1.16.2-7.2-nts-x64.zip

```text
[07:41:05] [DEBU] testing ext: mongodb
[07:41:05] [DEBU] Running command with result: /www/static-php-cli/buildroot/bin/php --ri "mongodb"
   SPC\exception\RuntimeException
  extension mongodb failed compile check
  at src/SPC/builder/traits/UnixBuilderTrait.php:74
     70▕             foreach ($this->exts as $ext) {
     71▕                 logger()->debug('testing ext: ' . $ext->getName());
     72▕                 [$ret] = shell()->execWithResult(BUILD_ROOT_PATH . '/bin/php --ri "' . $ext->getDistName() . '"', false);
     73▕                 if ($ret !== 0) {
  ➜  74▕                     throw new RuntimeException('extension ' . $ext->getName() . ' failed compile check');
     75▕                 }
     76▕
     77▕                 if (file_exists(ROOT_DIR . '/src/globals/tests/' . $ext->getName() . '.php')) {
     78▕                     // Trim additional content & escape special characters to allow inline usage
  1   src/SPC/builder/linux/LinuxBuilder.php:234
      SPC\builder\linux\LinuxBuilder::sanityCheck()
  2   src/SPC/command/BuildCliCommand.php:90
      SPC\builder\linux\LinuxBuilder::buildPHP()
[07:41:05] [CRIT] You can report this exception to static-php-cli GitHub repo.
```

**Comment by crazywhalecc** [comment]
Please provide the compilation command you used. 

`Failed compile check` is usually caused by the extension not being successfully added to the compilation. It is recommended to delete the `source/` directory before compiling.

**Comment by MrZ9001** [comment]
> Please provide the compilation command you used.
> 
> `Failed compile check` is usually caused by the extension not being successfully added to the compilation. It is recommended to delete the `source/` directory before compiling.

Command: `bin/spc build --build-cli "mongodb" --debug`
Local PHP Version: `8.2.11`
Local PHP extension:
```text
[PHP Modules]
calendar
Core
ctype
date
exif
FFI
fileinfo
filter
ftp
gettext
hash
iconv
json
libxml
mongodb
openssl
pcntl
pcre
PDO
Phar
posix
random
readline
Reflection
session
shmop
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
zlib

[Zend Modules]
```
Delete the `source/` directory and perform compilation again, still displaying `extension mongodb failed compile check`.

**Comment by crazywhalecc** [comment]
Fixed in #227 . You can clone the latest main branch or download `spc` binary from [actions](https://github.com/crazywhalecc/static-php-cli/actions/runs/6516823900)
